kqueen_ui/asset/dynamic/scss/main.scss

```diff
@@ -105,6 +105,15 @@ a, input[type="checkbox"] {
   }
 }
 
+.editable-file {
```

Comment:
I suggest to move from Global to Form section of the current file

Reply:
Done